back/src/database/schema.prisma

```diff
@@ -64,32 +61,44 @@ model UserProfile {
     id           Int               @id @unique @default(autoincrement())
     xp           Int               @default(0)
     elo          Int               @default(1000)
+    wonMatches   Int               @default(0)
+    lostMatches  Int               @default(0)
     picture      String
     achievements UserAchievement[]
     user         User?
 }
 
 model UserAchievement {
-    id            Int             @id @unique @default(autoincrement())
+    id            Int         @id @unique
```

Comment:
L'enum me paraissait mieux pour éviter les erreurs sur les id des achievments. D ailleurs la ta colonne en `@id` est pas vraiment un id ni unique vu que plusieurs joueurs peuvent avoir le meme achievment. Je vois plutot un truc avec un id en auto increment et un type qui est lié a l enum de la liste des achievments

Reply:
Elle est unique. Elle veut dire que tu ne peux pas avoir deux meme id au sein de cette meme table, c'est logique car l'user ne peut pas debloquer deux fois le meme achievement.
L'idee etait dans la BDD, n'enregistrer uniquement l'ID de l'achievement debloque par user

Reply:
En effet, je me suis fait bait par l interface de github MDR

Reply:
Certes un user ne peut pas débloquer 2 fois le même achievement mais 2 users peuvent débloquer le même. Genre si "jouer ta premier partie" est à l id 1, que je le débloque et toi aussi il y aura 2 fois 1 comme is et ça va casser

Reply:
Ah oui bien sur, je changerai ça !